python/nav/ipdevpoll/plugins/entity.py

```diff
@@ -51,12 +51,9 @@ def on_plugin_load(cls):
     @defer.inlineCallbacks
     def handle(self):
         self._logger.debug("Collecting physical entity data")
-        need_to_collect = yield self._need_to_collect()
-        # if need_to_collect:
-        if True:
-            physical_table = yield self.entitymib.get_entity_physical_table()
-            self._logger.debug("found %d entities", len(physical_table))
-            self._process_entities(physical_table)
+        physical_table = yield self.entitymib.get_entity_physical_table()
+        self._logger.debug("found %d entities", len(physical_table))
+        self._process_entities(physical_table)
```

Comment:
I would add a comment here noting that there used to be a hedge. @lunkwill42?

Reply:
Yeah, something needs to be done here.  I think the reason it was commented out was that I wasn't sure what to do.  The `ENTITY-MIB` provides a timestamp that tells when change was made to the table of physical entities.  It was used to avoid a full re-collection of the table if nothing had changed since the last collection.

However, there were some situations where the table was never updated again (which may be particular to some device types), which lead to this being commented out in lieu of a better solution.

If removed or commented out, the `_need_to_collect()` method might also be flagged as not referenced anywhere, causing someone to also delete that.

We could keep the whole thing commented out, or we could strip everything out, but add it back in as a Draft PR to flag the need to work on this issue later.  Thoughts?